src/plugins/hetero/executable_network.cpp

```diff
@@ -6,7 +6,7 @@
 #include "ie_metric_helpers.hpp"
 #include "executable_network.hpp"
 #include "async_infer_request.hpp"
-#include "itt.hpp"
+#include "src/itt.hpp"
```

Comment:
Please add include dir to cmake target instead of this `src/...`

Also you can create a separate PR and move all plugin files to src folder.